modules/lwjgl/core/src/templates/kotlin/core/windows/WindowsTypes.kt

```diff
@@ -631,4 +633,28 @@ val INPUT = struct(Module.CORE_WINDOWS, "INPUT") {
         HARDWAREINPUT("hi", "the information about a simulated hardware event")
     }("DUMMYUNIONNAME", "")
 }
-val LPINPUT = typedef(INPUT.p, "PINPUT")
\ No newline at end of file
+val LPINPUT = typedef(INPUT.p, "PINPUT")
+
+val DATA_BLOB = struct(Module.CORE_WINDOWS, "DATA_BLOB") {
+    documentation =
+        """
+        The {@code DATA_BLOB} structure contains an arbitrary array of bytes. The structure definition includes aliases appropriate to the various functions
+        that use it.    
+        """
+
+    AutoSize("pbData")..DWORD("cbData", "the count, in bytes, of data")
+    BYTE.p("pbData", "a pointer to the data buffer")
+}
+
+val CRYPTPROTECT_PROMPTSTRUCT = struct(Module.CORE_WINDOWS, "CRYPTPROTECT_PROMPTSTRUCT") {
+    documentation =
+        """
+        The {@code CRYPTPROTECT_PROMPTSTRUCT} structure provides the text of a prompt and information about when and where that prompt is to be displayed when
+        using the #CryptProtectData() and #CryptUnprotectData() functions.
+        """
+
+    DWORD("cbSize", "the size, in bytes, of this structure")
```

Comment:
Can we autosize this somehow?

Reply:
Yeah, with `Expression("SIZEOF")..`. Adds the `cbSize$Default()` overload, similar to the Vulkan bindings.